src/CloudNative.CloudEvents.NewtonsoftJson/JsonEventFormatter.cs

```diff
@@ -13,6 +13,8 @@
 
 namespace CloudNative.CloudEvents.NewtonsoftJson
 {
+    // TODO: Rename to JsonCloudEventFormatter? NewtonsoftJsonCloudEventFormatter?
```

Comment:
Yeah we may want to differentiate in the type name if we think apps might want both packages.

Reply:
I suspect that will be rare, to be honest - and apps can always use aliases etc. I'll leave the TODO there, but it's more something to revisit than "we definitely need to change this".